src/modules/logger/logged_topics.cpp

```diff
@@ -172,6 +173,7 @@ void LoggedTopics::add_high_rate_topics()
 	add_topic("manual_control_setpoint");
 	add_topic("rate_ctrl_status", 20);
 	add_topic("sensor_combined");
+	add_topic("tunnel");
```

Comment:
How about a new option just for tunnel? High-rate is really for analyzing vehicle flight and controller performance, we shouldn't overload it with other messages unless it's useful in that context.